### PR TITLE
Fix bugs of quick-x module

### PR DESCRIPTION
**Pull request by SunLightJuly**

### CODE REVIEW

File: lua/quick/LuaNodeManager.h [hunk 1]
```diff
@@ -32,7 +32,7 @@ THE SOFTWARE.
 
 NS_CC_BEGIN
 
-class CC_DLL LuaNodeManager : public Ref
+class LuaNodeManager : public Ref
```
Comment (minggo) [inline lua/quick/LuaNodeManager.h:35]: Why remove `CC_DLL`?

Comment (SunLightJuly) [inline lua/quick/LuaNodeManager.h:35]: If used prefix "CC_DLL", to build windows' project  will cause link error.

Comment (minggo) [inline lua/quick/LuaNodeManager.h:35]: But it can not be used in other libs if you don't export it.

Comment (SunLightJuly) [inline lua/quick/LuaNodeManager.h:35]: Yes. These classes are only used in lib "libluacocos2d". We don't need to export them.

**Pull request merged**